src/main/resources/application.properties

```diff
@@ -2,7 +2,12 @@ spring.application.name=user-data-store
 
 spring.jpa.hibernate.ddl-auto=none
 spring.jpa.open-in-view=false
+spring.jpa.properties.hibernate.connection.provider_disables_autocommit=true
```

Comment:
I have one concern: shouldn't this setting be applied across all backends? Not sure we want to do it differently in each project.

Reply:
I would recommend that. See https://vladmihalcea.com/why-you-should-always-use-hibernate-connection-provider_disables_autocommit-for-resource-local-jpa-transactions/

Reply:
Either we should remove the setting here or file issues to change it in all backends. I don't like the state when it is fixed in one application and stays original in others.

Reply:
Ok, agreed, let's test it here. After that revert or introduce it to all components. I have opened an internal issue https://github.com/orgs/wultra/projects/29/views/1?pane=issue&itemId=21830474